addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo><data>
+
+    <template id="event_booth_layout" name="Event Booth Layout">
+        <t t-call="website_event.layout">
+            <div class="o_wevent_booth_registration" name="event" t-att-data-event-id="event.id"
+                 itemscope="itemscope" itemtype="http://schema.org/Event">
+                <t t-call="website.record_cover">
+                    <t t-set="_record" t-value="event"/>
+                    <t t-set="use_filters" t-value="True"/>
+                    <t t-set="use_text_align" t-value="True"/>
+                    <t t-set="additionnal_classes" t-value="'pb128'"/>
+
+                    <div class="container d-flex flex-column flex-grow-1">
+                        <h1 class="text-white my-5">Get A Booth</h1>
+                        <div t-if="not event_booth_ids" class="alert alert-info">
+                            <span>This event is not open to exhibitors registration,
+                                <span t-if="request.env.user.has_group('event.group_event_manager')">
+                                    you can
+                                    <a class="text-nowrap" t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
+                                        <i class="fa fa-gear mr-1" role="img" aria-label="Configure" title="Configure event booths"/><em>Configure Booths</em>
+                                    </a>
+                                    for this event.
+                                </span>
+                                <span t-else="">
+                                    check our <a href="/event">list of future events</a>.
+                                </span>
+                            </span>
+                        </div>
+                        <div t-elif="event.is_finished" class="alert alert-info">
+                            <span>This event is finished. It's no longer possible to book a booth.</span>
+                        </div>
+                    </div>
+                </t>
+                <section class="mt-n5">
+                    <div class="container overflow-hidden">
+                        <div class="row mb64 bg-white no-gutters rounded shadow-sm">
+                            <t t-out="0"/>
+                        </div>
+                    </div>
+                </section>
+            </div>
+        </t>
+    </template>
+
+    <template id="event_booth_registration" name="Event Booth Registration">
+        <t t-call="website_event_booth.event_booth_layout">
+            <t t-if="event_booth_ids and not event.is_finished">
+                <div class="col-lg-9 p-4">
+                    <form method="post" class="form-horizontal justify-content-center mt32 js_website_submit_form"
```

Comment:
Just learned existence of this ``js_website_submit_form`` class, seems cool to have that kind of small feature available :)